.gitmodules

```diff
@@ -5,7 +5,7 @@
 	path = src/3rdparty/rpmalloc/rpmalloc
 	url = https://github.com/mjansson/rpmalloc.git
 [submodule "plugins/zynaddsubfx/zynaddsubfx"]
-	path = plugins/zynaddsubfx/zynaddsubfx
+	path = plugins/ZynAddSubFx/zynaddsubfx
```

Comment:
Should the case of these paths match?

Reply:
They would matter on Linux/Mac, but I think on Windows it ignores case. Good catch!

You'll also wants to updated the `[submodule "plugins/ZynAddSubFx/zynaddsubfx"]` section header.